NSF POSE announcement (#355)
unitaryfund/unitary.fund: src/content/blog/2023_mitiq_nsf_pose.md

---
title: NSF funding for the Mitiq open source ecosystem
author: Nathan Shammah and Will Zeng
day: 21
month: 8
year: 2023
---

We are thrilled to announce that Unitary Fund has been awarded a $1.5M [grant](https://www.nsf.gov/awardsearch/showAward?AWD_ID=2303643&HistoricalAwards=false) by the National Science Foundation (NSF) to grow the open-source ecosystem around the [Mitiq](https://mitiq.readthedocs.io/en/latest/) project. 

> Mitiq is a compiler that makes programs more robust to errors in quantum computers. Because quantum computers have high error rates, compiling for error robustness is critical for useful applications. Among other applications, Mitiq has been used to better simulate correlated materials, to increase the resolution of the time evolution of quantum systems out of equilibrium, and to increase the effective quantum volume of quantum processors - a key benchmark. The Mitiq OSE will attract and support a large community of users and contributors that extends beyond today?s core developers and initial user base. This expansion will include product management, infrastructure engineering, community management, governance, and user success. Specifically, this project will focus on user-led development of digital interfaces and resources, accelerated advancement in quantum error mitigation, compatibility with emerging hardware, dependencies and related software packages, establishment of standards and benchmarks, and dissemination of results and software integration. By building the Mitiq OSE, Shammah and Zeng will increase open access to software based techniques that reduce noise sensitivities of programs and protocols, accelerating the adoption of cutting edge research by quantum computing users and improving quantum technology for all users across a significant new industry.

Author: ```suggestion
## About Mitiq
[Mitiq](https://mitiq.readthedocs.io/en/latest/) is a compiler that makes programs more robust to errors in quantum computers. Because quantum computers have high error rates, compiling for error robustness is critical for useful applications. 

## About the Mitiq Open Source Ecosystem (OSE)
The Mitiq OSE will attract and support a large community of users and contributors that extends beyond today's core developers and initial user base. This expansion will include product management, infrastructure engineering, community management, governance, and user success. Specifically, this project will focus on user-led development of digital interfaces and resources, accelerated advancement in quantum error mitigation, compatibility with emerging hardware, dependencies and related software packages, establishment of standards and benchmarks, and dissemination of results and software integration. 

By building the Mitiq OSE, we will increase open access to software based techniques that reduce noise sensitivities of programs and protocols, accelerating the adoption of cutting edge research by quantum computing users and improving quantum technology for all users across a significant new industry.
```
